README.md

```diff
@@ -40,109 +116,222 @@ You can download the script from the following locations:
+* mscs-client-location - Assign the location of the client .jar file.
+* mscs-server-version - Assign the version of the server software.
+* mscs-server-jar - Assign the .jar file for the server software.
+* mscs-server-url - Assign the download URL for the server software.
+* mscs-server-args - Assign the arguments to the server.
+* mscs-initial-memory - Assign the initial amount of memory for the server.
+* mscs-maximum-memory - Assign the maximum amount of memory for the server.
+* mscs-server-location - Assign the location of the server .jar file.
+* mscs-server-command - Assign the command to run for the server.
+
+The following variables may be used in some of the values of the above keys:
+* $JAVA - The Java virtual machine.
+* $CURRENT_VERSION - The current Mojang Minecraft release version.
+* $CLIENT_VERSION - The version of the client software.
+* $SERVER_VERSION - The version of the server software.
+* $SERVER_JAR - The .jar file to run for the server.
+* $SERVER_ARGS - The arguments to the server.
+* $INITIAL_MEMORY - The initial amount of memory for the server.
+* $MAXIMUM_MEMORY - The maximum amount of memory for the server.
+* $SERVER_LOCATION - The location of the server .jar file.
+
+#### Example key/value pairs
+
+    mscs-enabled=true
+    mscs-version-type=release
+    mscs-client-version=$CURRENT_VERSION
+    mscs-client-jar=$CLIENT_VERSION.jar
+    mscs-client-url=https://s3.amazonaws.com/Minecraft.Download/versions/$CLIENT_VERSION/$CLIENT_VERSION.jar
+    mscs-client-location=/opt/mscs/client/$CLIENT_VERSION
+    mscs-server-version=$CURRENT_VERSION
+    mscs-server-jar=minecraft_server.$SERVER_VERSION.jar
+    mscs-server-url=https://s3.amazonaws.com/Minecraft.Download/versions/$SERVER_VERSION/minecraft_server.$SERVER_VERSION.jar
+    mscs-server-args=nogui
+    mscs-initial-memory=128M
+    mscs-maximum-memory=2048M
+    mscs-server-location=/opt/mscs/server
+    mscs-server-command=$JAVA -Xms$INITIAL_MEMORY -Xmx$MAXIMUM_MEMORY -jar $SERVER_LOCATION/$SERVER_JAR $SERVER_ARGS
+
+Run a Minecraft version 1.6.4 server:
+
+    mscs-client-version=1.6.4
+    mscs-server-version=1.6.4
+
+Use Forge to run a 1.8.4 server (requires additional setup):
+
+    mscs-client-version=1.8.4
+    mscs-server-version=1.8.4
+    mscs-server-jar=forge-1.8-11.14.1.1419-universal.jar
+    mscs-server-url=http://files.minecraftforge.net/maven/net/minecraftforge/forge/1.8-11.14.1.1419/forge-1.8-11.14.1.1419-universal.jar
+
+Use the latest BungeeCord successful build (requires additional setup):
 
+    mscs-server-jar=BungeeCord.jar
+    mscs-server-url=http://ci.md-5.net/job/BungeeCord/lastSuccessfulBuild/artifact/bootstrap/target/BungeeCord.jar
 
-## Usage
+##### Additional documentation
+
+More examples and documentation on server customization can be found on the [wiki](https://github.com/sandain/MinecraftServerControlScript/wiki/Server-Customization-Examples) page.
+
+### The msctl file
+The `msctl` file can be found at `/usr/local/bin`.
```

Comment:
You should not be editing this file. There is a section in `msctl` that will load config from other locations.
Locations include `$HOME/mscs.conf` `$HOME/.config/mscs/mscs.conf`, or `/etc/default/mscs` (mscs set this value and calls `msctl`)
```
# Override Global Variables
# ---------------------------------------------------------------------------
# Override global variables below by adding them to the specified file.  If
# the file does not exist, the default values below will be used.
# Configuration files are checked in the following order. First file found is
# sourced and the rest are discarded:
#   command line option "-c".
#   $HOME/mscs.conf
#   $HOME/.config/mscs/mscs.conf
if [ -r "$MSCS_DEFAULTS_CL" ]; then
  MSCS_DEFAULTS="$MSCS_DEFAULTS_CL"
elif [ -r "$HOME/mscs.conf" ]; then
  MSCS_DEFAULTS="$HOME/mscs.conf"
elif [ -r "$HOME/.config/mscs/mscs.conf" ]; then
  MSCS_DEFAULTS="$HOME/.config/mscs/mscs.conf"
fi
```